doc/api_methods/UpdateList.md

```diff
@@ -0,0 +1,38 @@
+[back to list](../Readme.md)
+
+# Add Subscriber
+
+## `array updateList(array $list)`
+
+This method provides functionality for updating a list name or description.
+
+It returns the updated list. See [Get Lists](GetLists.md) for a list data structure description.
+
+## Arguments
+
+### `$list` (required)
+
+An associative array which contains list data.
+
+| Property               | Type         | Limits    | Description                |
+| ---------------------- | ------------ | --------- | -------------------------- |
+| id (required)          | string       | 11 chars  | A id of the list.          |
```

Comment:
I got a type error when using a string:

`wp> \MailPoet\API\API::MP('v1')->updateList(['id'=>'2','name'=>'superWoo'])
TypeError: MailPoet\Segments\SegmentsRepository::createOrUpdate(): Argument #5 ($id) must be of type ?int, string given,`